sdk/core/azure-core/src/main/java/com/azure/core/util/Header.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+import java.util.Objects;
+
+/**
+ * It represents a single header for a request being sent to Azure Service.
```

Comment:
```suggestion
 * Represents a single header for a request.
```

Can drop the Azure bit as it isn't explicitly required for our SDKs to talk to Azure services.